node_modules/col-core/lib/db.js

```diff
@@ -78,19 +78,323 @@ var init = module.exports.init = function(callback) {
+   * communicate with. It holds all the information that's needed to interact with the
+   * Canvas API and to embed on of the Collabosphere tools into Canvas
    *
-   * @property  {Number}      id                      The id of the course
+   * @property  {String}    app_domain      The domain where canvas is running on
+   * @property  {String}    api_key         The key that can be used to interact with the Canvas API
+   * @property  {String}    lti_key         The basic LTI key that will be used to embed the tools into Canvas
+   * @property  {String}    lti_secret      The basic LTI secret that will be used to embed the tools into Canvas
+   * @property  {Boolean}   use_https       Whether the Canvas API is running on https
    */
-  var Course = module.exports.Course = sequelize.define('Course', {
-    'id': {
-      'type': Sequelize.INTEGER,
+  var Canvas = module.exports.Canvas = sequelize.define('canvas', {
+    'app_domain': {
+      'type': Sequelize.STRING,
       'allowNull': false,
-      'unique': true,
       'primaryKey': true
+    },
+    'api_key': {
+      'type': Sequelize.STRING,
+      'allowNull': false
+    },
+    'lti_key': {
+      'type': Sequelize.STRING,
+      'allowNull': false,
+      'unique': true
+    },
+    'lti_secret': {
+      'type': Sequelize.STRING,
+      'allowNull': false,
+      'unique': true
+    },
+    'use_https': {
+      'type': Sequelize.BOOLEAN,
+      'defaultValue': true
+    }
+  }, {
+    'underscored': true
+  });
+
+  /**
+   * The `course` table keeps track of each course where the tools have been embedded in
+   *
+   * @property  {Number}      canvas_course_id        The id of the course in Canvas
+   * @property  {Boolean}     enable_upload           Whether or not students are allowed to upload assets directly
+   * @property  {Boolean}     enable_sync             Whether or not this course's data should be synced with Canvas
+   */
+  var Course = module.exports.Course = sequelize.define('course', {
+    'canvas_course_id': {
+      'type': Sequelize.INTEGER,
+      'allowNull': false
+    },
+    'enable_upload': {
+      'type': Sequelize.BOOLEAN,
+      'allowNull': false,
+      'defaultValue': true
+    },
+    'enable_sync': {
+      'type': Sequelize.BOOLEAN,
+      'allowNull': false,
+      'defaultValue': true
+    }
+  }, {
+    'underscored': true
+  });
+
+  // Each course belongs to a specific Canvas instance
+  Course.belongsTo(Canvas, {'onDelete': 'CASCADE'});
+
+  /**
+   * The `user` table keeps track of each user that is enrolled in a course. This means that
+   * the same physical student might end up with two student records within Collabosphere. As
+   * there's no overlap between courses, this is acceptable.
+   *
+   * @property  {Number}        canvas_user_id          The id of the user in Canvas
+   * @property  {String}        canvas_course_role      The role of the user in the course
```

Comment:
I didn't realise that Canvas provides it as a comma-separated string. This makes sense.